Abennat/DLS001.xml

```diff
@@ -51,7 +52,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
             </langUsage>
         </profileDesc>
         <revisionDesc>
-            <change who="MA" when="2022-04-07">Created entity</change>
+            <change who="MA" when="2022-04-07">Created entity
```

Comment:
dear Addisie, now there is no closing </change> tag at the end of line 55 (it was there, but you deleted it)